gsoc/2020/mentors.md

```diff
@@ -7,9 +7,12 @@ layout: plain
 
 * Christian Ariza [christian.ariza@gmail.com](mailto:christian.ariza@gmail.com) CERN
 * Vasco Chibante Barroso [vmcb@cern.ch](mailto:vmcb@cern.ch) CERN
+* Brian Bockelman [bbockelman@morgridge.org](mailto:bbockelman@morgridge.org) Morgridge Institute for Research
```

Comment:
Alphabetical order

Reply:
:) I thought Barroso is before Bockelman ?

Reply:
I checked in CERN phonebook what is a first/second name, and I agree with your comment (I fixed order).